python/ray/utils.py

```diff
@@ -268,3 +268,28 @@ def merge_dicts(d1, d2):
     d = d1.copy()
     d.update(d2)
     return d
+
+
+def check_oversized_pickle(pickled, name, obj_type, worker):
+    """Send a warning message if the pickled object is too large.
+
+    Args:
+        pickled: the pickled object.
+        name: name of the pickled object.
+        obj_type: type of the pickled object, can be 'function', 'actor',
+            or 'object'.
+        worker: the worker used to send warning message.
+    """
+    length = len(pickled)
+    if length <= ray_constants.PICKLE_OBJECT_WARNING_SIZE:
+        return
+    warning_message = (
```

Comment:
There should be an extra space at the end of the first two lines so that there is a space when the strings are concatenated together.